innersource-license.md

```diff
@@ -0,0 +1,83 @@
+## Title
+
+InnerSource License
+
+## Patlet
+
+Two companies that belong to the same group want to share software source code with each other but they are concerned about the implications in terms of legal liabilities or cross-company accounting. An **InnerSource License** provides a reusable legal framework for the inter-company agreement within the group, which opens up new collaboration options.
+
+## Problem
+
+When two companies within a group want to share code with each other, they need an agreement about the terms and often a legal contract. Creating such agreements on a per project basis takes effort and creates a barrier for sharing. i.e. a team/company might decide not to share their source code with another company in the group because it seems complicated.
+
+Barriers for sharing can lead to silos and duplication of effort in rebuilding similar solutions in multiple parts of the group.
+
+At the time of sharing the source code, it can not be reliably predicted what the value of sharing will be. Hence if the activity of sharing has any significant cost, the companies are unlikely to do it, as they are concerned about the return on investment.
+
+## Context
+
+- A large group with many companies (subsidiaries) that want to share code. (when the group gets larger, the value of this pattern increases)
+- These companies are legally independent units.
+- Multiple of these companies/subsidiaries are developing software, and are using services of the other companies. They have a motivation to contribute to each other’s source code.
+- Certain size of developers (crowdsourcing and ecosystem effects of InnerSource)
+
+## Forces
+
+- **Level of effort** required to write formal agreements, especially if they need to take into account technical, legal, and business perspectives.
+- Enterprises have many **internal regulations,** any agreements have to comply with these regulations, e.g. security, privacy, procurement processes, etc. The volume of regulations can make it difficult to assess whether sharing software between two companies is compliant with these regulations, especially when there is no standard procedure.
```

Comment:
`Internal regulation` are one thing. Another thing are plain legal requirements. Did you interviews yield any insights on this (potential) additional force?

Reply:
We did not go deeper into this. The **internal regulations** aspect seemed to be the more pressing one.

@cornelius what do you say, do **external regulations** also impact this pattern? Should we add a bullet about **external regulations** to the **Forces**?

Reply:
I would say that external regulations do have an impact, but not a very specific one. Legal documents have to match plain legal requirements, but that's kind of a given.

Reply:
> I would say that external regulations do have an impact, but not a very specific one. 

I assume you worked a lot with our legal team in designing your org's InnerSource license. Would you be open to asking them if they see specific external regulations that require that? (I understand if you don't want to: questions to the legal team can leverage unintended consequences).

If in your observation the internal regulations were they key force. Please feel free to resolve this comment @cornelius and @spier.

Reply:
Given that 10 days have passed, I will resolve this. Can be expanded on later by other companies/contributors.